bigc/api_client.py

```diff
@@ -143,14 +145,16 @@ def request(self, method: str, path: str, **kwargs):
         response = super().request(method, path, **kwargs)
         return None if response is None else response['data']
 
-    def get_many(self, path: str, **kwargs) -> Generator:
+    def get_many(self, path: str, *, page_size: Optional[int] = None, **kwargs) -> Generator:
+        page_size = MAX_V3_PAGE_SIZE if page_size is None else int(page_size)
```

Comment:
Why `int(...)` `page_size`?

Reply:
I am assuming because the page size could be passed from the request as a string version of the int and this would convert it if that is the case.

Reply:
Then the type hint is incorrect

Reply:
If `page_size` was e.g. a `str` then the request would go through fine but it would blow up on line 133, I'd rather it blow up early. We technically don't need to do that in v3 but I'd rather it just be the same.

Reply:
I do not understand how that excuses the incorrect typehint but ok